### PR TITLE
bug/422_fix_grouping_rules_conf

### PR DESCRIPTION
**Pull request by frbattid**
* Fixes issue #422 
* No unit tests nor (unofficial) e2e tests required
* Assignee @iariasleon

### CODE REVIEW

File: conf/grouping_rules.conf.template [hunk 1]
```diff
@@ -14,37 +14,62 @@
...
 
-# This file defines a matching table for context data grouping. The table is composed of several mathcing rules. It
-# works by defining a set of fields within the Orion notification and a regular expression to be applied over the
-# concatenation of those fields. If there is a matching then the destination within the matching rule is used as the
-# name of the permanent storage for the data (it will be a file in the case of HDFS, a MySQL table or a CKAN resource).
+# This file defines a set of grouping rules for context data grouping. It works by defining a set of fields within the
+# Orion notification and a regular expression to be applied over the concatenation of those fields. If there is a
+# matching then:
+# - The destination within the rule is used as the name of the storage element (i.e. this will be a file in
+#   the case of HDFS, part of a MySQL table or a CKAN resource).
```
Comment (iariasleon) [inline conf/grouping_rules.conf.template]: append mongo/sth behavior. Ex: in collection

Comment (iariasleon) [inline conf/grouping_rules.conf.template]: append mongo/sth behavior. Ex: in collection

File: conf/grouping_rules.conf.template [hunk 2]
```diff
@@ -14,37 +14,62 @@
...
 #
-# The available "dictionary" of fields is:
-#  - entitydId
-#  - entityType
-#  - servicePath
+# Being:
+# - id: A unique unsigned integer-based identifier. Not really used in the current implementation, but could be useful
+#   in the future.
+# - fields: These are the fields that will be concatenated for regular expression matching. The available dictionary of
+#   fields for concatenation is "entityId", "entityType" and "fiwareService". The order of these fields is important
```
Comment (iariasleon) [inline conf/grouping_rules.conf.template]: "fiwareService" or "fiwareServicePath" ?

File: conf/grouping_rules.conf.template [hunk 3]
```diff
@@ -14,37 +14,62 @@
...
+#   since the concatenation is made from left to right.
+# - regex: Java-like regular expression to be applied on the concatenated fields.
+# - destination: Name of the HDFS file, MySQL table or CKAN resource where the data will be effectively persisted.
```
Comment (iariasleon) [inline conf/grouping_rules.conf.template]: append mongo and sth

File: conf/grouping_rules.conf.template [hunk 4]
```diff
@@ -14,37 +14,62 @@
...
+#   Please, have a look to doc/design/naming_conventions.md for more details.
+# - fiware_service_path: New fiware-servicePath replacing the notified one. The sinks will translate this into the name
+#   of the HDFS folder or CKAN package where the above destination entity will be placed. In the case of MySQL, this
+#   prefixes the table name. Please, have a look to doc/design/naming_conventions.md for more details.
```
Comment (iariasleon) [inline conf/grouping_rules.conf.template]: append mongo and sth behaviors

File: conf/grouping_rules.conf.template (hunk 2 above)
Comment (frbattid) [inline conf/grouping_rules.conf.template]: Nice catch, it is the servicePath!!

File: conf/grouping_rules.conf.template (hunk 1 above)
Comment (frbattid) [inline conf/grouping_rules.conf.template]: Fixed in https://github.com/telefonicaid/fiware-cygnus/commit/ec3abfa8b762ab7f947c7d74f3fb62dbdaa1722a

File: conf/grouping_rules.conf.template (hunk 2 above)
Comment (frbattid) [inline conf/grouping_rules.conf.template]: Fixed in https://github.com/telefonicaid/fiware-cygnus/commit/ec3abfa8b762ab7f947c7d74f3fb62dbdaa1722a

File: conf/grouping_rules.conf.template (hunk 3 above)
Comment (frbattid) [inline conf/grouping_rules.conf.template]: Fixed in https://github.com/telefonicaid/fiware-cygnus/commit/ec3abfa8b762ab7f947c7d74f3fb62dbdaa1722a

File: conf/grouping_rules.conf.template (hunk 4 above)
Comment (frbattid) [inline conf/grouping_rules.conf.template]: Fixed in https://github.com/telefonicaid/fiware-cygnus/commit/ec3abfa8b762ab7f947c7d74f3fb62dbdaa1722a

**Comment by iariasleon** [comment]
LGTM

**Pull request merged**